### PR TITLE
New gas customize feature test updates

### PR DESCRIPTION
**Pull request by danjm**
This PR adds some updates to e2e tests and unit tests to the gas customization feature branch

### CODE REVIEW

File: test/e2e/beta/metamask-beta-ui.spec.js [hunk 1]
```diff
@@ -271,17 +284,17 @@ describe('MetaMask', function () {
       await driver.wait(until.stalenessOf(accountModal))
       await delay(regularDelayMs)
     })
-    it('show account details dropdown menu', async () => {
+    // it('show account details dropdown menu', async () => {
```
Comment (alextsg) [inline test/e2e/beta/metamask-beta-ui.spec.js]: Should we just remove these lines or add a comment on why they've been commented out?

File: ui/app/ducks/gas.duck.js [hunk 2]
```diff
@@ -207,10 +207,10 @@ export function fetchGasEstimates (blockTime) {
       priceAndTimeEstimatesLastRetrieved,
       priceAndTimeEstimates,
     } = getState().gas
-    const timeLastRetrieved = priceAndTimeEstimatesLastRetrieved || loadLocalStorageData('GAS_API_ESTIMATES_LAST_RETRIEVED')
+    const timeLastRetrieved = priceAndTimeEstimatesLastRetrieved || loadLocalStorageData('GAS_API_ESTIMATES_LAST_RETRIEVED') || 0
 
     dispatch(gasEstimatesLoadingStarted())
-
+    console.log('Date.now() - timeLastRetrieved', Date.now() - timeLastRetrieved)
```
Comment (alextsg) [inline ui/app/ducks/gas.duck.js]: Remove `console.log`

File: test/e2e/beta/metamask-beta-ui.spec.js (hunk 1 above)
Comment (danjm) [reply test/e2e/beta/metamask-beta-ui.spec.js]: They were moved to a separate file in my PR that created e2e tests for the extension view. I just rebased, so they are gone now.

File: ui/app/ducks/gas.duck.js (hunk 2 above)
Comment (danjm) [reply ui/app/ducks/gas.duck.js]: removed

**Review by whymarrh — APPROVED** [review]

**Pull request merged**